docs/config/README.md

```diff
@@ -4528,3 +4528,33 @@ detect_extensions = ['pst'] # filters *.pst files
 shell = ['pwsh.exe', '-NoProfile', '-Command']
 use_stdin = false
 ```
+
+## Random
+
+`random` modules randomise their output every time the prompt is displayed.
+
+### Options
+
+| Option        | Default              | Description                          |
+| ------------- | -------------------- | ------------------------------------ |
+| `styles`      | `['>', '$', '%']`    | List of style variables to pick from |
+| `symbols`     | `['bold', 'italic']` | List of symbols to pick from         |
```

Comment:
The option column doesn't match the default and description columns.

Reply:
I'm not sure what you mean?

Reply:
I'm not sure what you mean?

Reply:
Are `['>', '$', '%']` styles or symbols?

Reply:
Oh My God :woman_facepalming:  :woman_facepalming:  :woman_facepalming: well spotted I guess xD

Reply:
You're welcome.